WordPress/Classes/Models/ReaderTeamTopic.swift

```diff
@@ -11,6 +11,10 @@ import Foundation
         return slug == ReaderTeamTopic.a8cSlug ? .readerA8CShown : .readerP2Shown
     }
 
+    var organizationType: SiteOrganizationType {
```

Comment:
Wfm.  However, if we wanted to have consistency with ReaderSiteTopic we could use the organization_id here also.  It's already returned by the /read/menu endpoint, we'd just need to add it to RemoteReaderTopic and then to the ReaderTeamTopic model. Not sure if it's worth the effort.  Up to you!

Reply:
In this particular case, I don't think it's worth it. But it probably wouldn't hurt for consistency, clarity, etc. I'll do it in a follow up PR.